src/assistant/Conversations/Sources/SourcesItem.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+import { useClient, generateWebLink } from 'cozy-client'
+import { isNote } from 'cozy-client/dist/models/file'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+
+import { getDriveMimeTypeIcon } from 'assistant/Search/getIconForSearchResult'
+
+import styles from './styles.styl'
+
+const SourcesItem = ({ file }) => {
+  const client = useClient()
+
+  const docUrl = generateWebLink({
+    slug: isNote(file) ? 'notes' : 'drive',
+    cozyUrl: client?.getStackClient().uri,
+    subDomainType: client?.getInstanceOptions().subdomain,
+    hash: isNote(file)
+      ? `/n/${file._id}`
+      : `/folder/${file.dir_id}/file/${file._id}`
+  })
+
+  return (
+    <ListItem
```

Comment:
We should a link component here with `target=_blank` so user take advantage of the browser's default functionality (e.g. accessibility). I see two options : 
```jsx
<ListItem>
  <a href={docUrl}>{...icon + text }</a>
</ListItem>
```
which would render in html ul > li > a
```jsx
<ListItem component="a" href={docUrl}>
  {...icon + text }
</ListItem>
```
Which would render ul > a. The last one is cleaner in React but html is better in the first one. Up to you

Reply:
en effet bien vu, généralement on utilise la deuxième approche